openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -2834,6 +2837,7 @@ private void incrementalChange() {
       updateInternal(false);
       incrementalChangeDescription = changeDescription;
       incrementalChangeDescription.setPreviousVersion(original.getVersion());
+      EntityRepository.this.incrementalChangeDescription = incrementalChangeDescription;
```

Comment:
what are we doing here  ? this doesn't even make any sense
we are keeping incremental change at entity repository 
level?